ndmg/stats/qa_tractography.py

```diff
@@ -0,0 +1,113 @@
+import nibabel as nib
+import numpy as np
+import matplotlib.pyplot as plt
+import os
+
+from fury import actor, window, colormap, ui
+from dipy.tracking.streamline import Streamlines
+from PIL import Image
+from matplotlib.colors import LinearSegmentedColormap
+
+def combine_plot(qa_out_path,brain_path):
+    """
+    Combine the nine plots produced by qa_tractography function into one image.
+
+    Parameters
+    ----------
+    qa_out_path : str
+        Path for the output QA imgae file(s)
+    brain_path : str
+        Path for the reference brain scan, in order to get the scan volume.
+    """
+    
+    brain_ref = nib.load(brain_path)
+    brain_ref_shape = brain_ref.shape[0:3]
+
+#   plt.rcParams.update({"axes.labelsize":"x-large","axes.titlesize":"x-large"})
+    plots_list = ['imgage1000000.png','imgage1000001.png','imgage1000002.png',
+                 'imgage2000000.png','imgage2000001.png','imgage2000002.png',
+                 'imgage3000000.png','imgage3000001.png','imgage3000002.png']
+    y_labels = ["Rotate with Z axis","Rotate with Y axis","Rotate with X axis"]
+    x_labels = ["angle=0","angle=120","angle=240"]
+    
+    #Temporary list to store the open image:
+    im=[] 
+    #Temporary list to store the image nparray:
+    im_arr=[] 
+
+    fig= plt.figure()
+    plt.title(f'QA_tractography. Scan volume = {brain_ref_shape} \n\n', fontsize=60,fontweight='bold')
+    plt.xticks([])
+    plt.yticks([])
+    plt.axis("off")
+
+    j = 0
+    for i in range(9):
+        im.append(Image.open(qa_out_path + "/"+ plots_list[i]))
+        im_arr.append(np.array(im[i])) 
+        #Change the background of the image into black:
+        im_arr[i] = np.where(im_arr[i]<=0.01, 255, im_arr[i]) 
+        ax = fig.add_subplot(3,3,i+1)
+        ax.imshow(im_arr[i],interpolation="none",alpha=0.9)
+        if i<3:
+            ax.set_title(x_labels[i],fontsize=60,fontweight='bold')
+        if i % 3 == 0:
+            ax.set_ylabel(y_labels[j],fontsize=60,fontweight='bold')
+            j = j + 1
+        plt.xticks([])
+        plt.yticks([])
+            
+        im[i].close()
+    fig.set_size_inches(40, 40, forward = True)
+    fig.savefig(qa_out_path + "/" + 'qa_tractography.png', format='png')
+
+    for plot in plots_list:
+        os.chdir(qa_out_path)
+        #print(os.getcwd())
+        if os.path.exists(qa_out_path + "/" + plot):
+            os.remove(plot)
+        else:
```

Comment:
This should throw an error if no files were generated, something is very wrong with the pipeline if no streamlines were found

Reply:
I'm not sure if I understand your meanings. But you can have a look of what I have changed.
https://github.com/neurodata/ndmg/pull/375/commits/f9edd4a64206da4025ac3ece5970c649b197ff73